src/compile/utils.ts

```diff
@@ -46,7 +50,7 @@ export class CompileFailedError extends Error {
 
 export type ImportFinder = (filePath: string) => { contents: string } | { error: string };
 
-export function getCompilerForVersion(version: string): any {
+export function getWasmCompilerForVersion(version: string): any {
```

Comment:
Need to revise why is this function still used. I recall that one of the compiler wrappers has same logic.

Reply:
Removed in https://github.com/ConsenSys/solc-typed-ast/pull/88